src/main/java/com/example/springbootbookshop/dto/CreateBookRequestDto.java

```diff
@@ -0,0 +1,14 @@
+package com.example.springbootbookshop.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class CreateBookRequestDto {
+    private String title;
+    private String author;
+    private String isbn;
+    private BigDecimal price;
+    private String description;
+    private String coverImage;
+}
```

Comment:
it is better to make it a record

Reply:
i switch to record and have error in this raw
 <configuration>
                    <configLocation>${maven.checkstyle.plugin.configLocation}</configLocation>
                    <encoding>UTF-8</encoding> ///error can`t read this property
                    <consoleOutput>true</consoleOutput>
                    <failsOnError>true</failsOnError>
                    <linkXRef>false</linkXRef>
                    <sourceDirectories>src</sourceDirectories>
                </configuration>